testdata/import/gz/BUILD

```diff
@@ -0,0 +1,11 @@
+load("//container:container.bzl", "container_import")
```

Comment:
Could you explain your reasoning why you think this is worth splitting off into a separate file?

Reply:
thanks for explaining, sgtm to keep them in separate folders. Could you create a single folder under testdata/ for each of these files and give them a name that is explicitly related to what is being declared (e.g., this second file could be testdata/pause_piecemeal_gz/BUILD, or something like that, as that the only thing it declares)

Reply:
thanks, see comment in testdata/import/BUILD